### DISCUSSION TITLE
Add http/https support

### DISCUSSION THREAD

**Discussion by allblue** [1Remote/1Remote#480] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
In routers, firewalls and other network devices, web management interface is more commonly used, and I hope to add http/https protocol support
在路由器、防火墙等网络设备中，web管理界面更为常用，希望能增加http/https协议支持

**Describe the solution you'd like**
Can be based on the system’s own edge or other browser kernel, open web pages in the software, and carry out one-stop configuration. If the software can save the username and password, it would be better.
可基于系统自带的edge或其他浏览器内核，在软件内打开网页，进行一站式配置。如果能保存用户名、密码更好

**Describe alternatives you've considered**
no

**Additional context**

**Comment by majkinetor** [comment]
Not sure what you mean with this, could you elaborate ?

**Reply by allblue** [reply]
For H3C switches, Topsec firewalls and other network devices, manufacturers prefer to use http/https+web pages for configuration management, rather than ssh. Therefore, I hope that 1remote can also support http/https session.

The implementation effect can refer to Remote Desktop Manager.
![image](https://github.com/1Remote/1Remote/assets/844763/3bbad27a-4a64-4a3b-be82-2062c08a07ba)

**Reply by majkinetor** [reply]
So, you want browser session?

If so, that is already possible, just use App tab with your browser and add starting link.

We can not inject password for the time being though, and I think it's probably out of the scope for this project, but you can use browser password management.

<details><summary>Screenshot</summary> 

![image](https://github.com/1Remote/1Remote/assets/85767/e2522d13-c82e-4f79-968b-0dac65e0e19f)



</details>

**Reply by allblue** [reply]
yeah, you can achieve a similar effect using the start-process command in powershell. But is it possible to integrate browser tabs into the premotem window, rather than just using premotem as an address book?

**Reply by majkinetor** [reply]
Yeah, there is an option _Hosting_ just enable it.

**Reply by allblue** [reply]
Unfortunately, I tested chrome and firefox, and the _hosting_ did not work.

**Reply by majkinetor** [reply, marked as answer]
I tested chrome and it did work

![image](https://github.com/1Remote/1Remote/assets/85767/f4bc0932-4752-4ac5-b0fd-76c685954574)

![image](https://github.com/1Remote/1Remote/assets/85767/b0f63516-0112-4d09-b12b-1f2b009d5f71)

**Reply by majkinetor** [reply]
Firefox didn't on the other hand.
IExplore works too.

**Reply by allblue** [reply]
My mistake, chrome portable version does not work, changed to the installation version is normal, thank you
